src/Microsoft.DotNet.Build.Tasks.Feed/BlobFeedAction.cs

```diff
@@ -246,7 +247,7 @@ private AzureFileSystem GetAzureFileSystem()
         {
             LocalSettings settings = GetSettings();
             AzureFileSystem fileSystem = GetAzureFileSystem();
-            bool result = await InitCommand.RunAsync(settings, fileSystem, true, true, new SleetLogger(Log), CancellationToken);
+            bool result = await InitCommand.RunAsync(settings, fileSystem, false, false, new SleetLogger(Log), CancellationToken);
```

Comment:
What are these bool parameters? Can you use named parameters there to help document them?